apps/shell/README.md

```diff
@@ -0,0 +1,80 @@
+# TASH - TinyAra SHell
+
+lightweight shell
+
+## How to enable
+
+Enable configuration of TASH through menuconfig
+```bash
+cd $TIZENRT_BASEDIR
+cd os
+make menuconfig
+```
+Select menu
+```bash
+Application Configuration -> Shell -> Enable shell to y
+```
+## How to register a command on TASH
+
+There are two methods, in code and in Makefile
+
+### Method in code
+
+TASH supports **tash_cmd_install** and **tash_cmdlist_install** functions.  
+The **tash_cmd_install** supports registering one command, but the **tash_cmdlist_install** does many commands by list.  
```

Comment:
I would like to suggest the following format here.

TASH has two public APIs to install commands :- 
* tash_cmd_install  <We can have the description of Public API here rather than having them below.
* tash_cmdlist_install